src/stub.js

```diff
@@ -0,0 +1,353 @@
+/* eslint-disable -- this is third party code */
```

Comment:
Do you know why this is still formatted?

Reply:
I think my editor must have formatted the file one save. I don't have a strong preference between minified and unminified stub code.